airbyte-webapp/src/pages/SettingsPage/pages/AccountPage/components/AccountForm.tsx

```diff
@@ -4,22 +4,25 @@ import { FormattedMessage, useIntl } from "react-intl";
 import styled from "styled-components";
 import * as yup from "yup";
 
-import { LoadingButton } from "components";
-import LabeledInput from "components/LabeledInput";
+import { Label, LabeledInput, LabeledSwitch, LoadingButton } from "components";
+import { InfoTooltip } from "components/base/Tooltip";
 import { Row, Cell } from "components/SimpleTableComponents";
 
+import { useAdvancedModeSetting } from "hooks/services/useAdvancedModeSetting";
+
+import styles from "./AccountForm.module.scss";
+
 const InputRow = styled(Row)`
   height: auto;
-  margin-bottom: 40px;
 `;
 
-const ButtonCell = styled(Cell)`
-  &:last-child {
-    text-align: left;
-  }
-  padding-left: 11px;
-  height: 9px;
-`;
+// const ButtonCell = styled(Cell)`
+//   &:last-child {
+//     text-align: left;
+//   }
+//   padding-left: 11px;
+//   height: 9px;
+// `;
```

Comment:
Should this be removed?

Reply:
:man_facepalming: Yyyyyep, right on it. Thanks for catching that!